### ISSUE TITLE
Add Emojis

### ISSUE THREAD

**Issue by CometWhoosh** [white-van/WVAnonBot#45]
### What problem are you facing?

Right now, users who don't have nitro can't send messages that contain emojis from the server the bot is in. 

### What can we do to fix that problem?

Allow users to send emojis from the server by typing in the name of the emoji enclosed in colons and have the bot replace it with the actual emoji (for example, ":blobheart:" would be replaced with the blobheart emoji).

In the case that users actually wanted to send the name of the emoji enclosed in servers, we can just let them escape it with a backslash (so "\:blobheart:" would just become ":blobheart:")

**Comment by jcserv** [comment]
As an avid emoji user, +1 +1 +1 😂

**Comment by LaZeAsh** [comment]
This wouldn't be possible for ALL of the emojis on discord because the bot has to be in the server to send that emoji, I think this is possible for wv though

**Comment by CometWhoosh** [comment]
Yep! Right now the bot is configured to just work with WV, so there shouldn't be a problem. But I think making it scalable so that it can work with other servers would be a great improvement in the future.

**Comment by LaZeAsh** [comment]
Just wanted to leave my though process here...

I think it would be cooler if we copied NQN (Not Quite Nitro) and made it so if someone says ":kek: funny" the bot deletes the message and resends a webhook in the same channel with the pfp of the og user and the name with the message

Dunno if that makes sense but if you want to look deeper into it just look into NQN or Not Quite Nitro
